cpp/src/Ice/OutgoingAsync.cpp

```diff
@@ -691,19 +691,31 @@ OutgoingAsync::response()
                 {
                     case replyObjectNotExist:
                     {
-                        throw ObjectNotExistException{__FILE__, __LINE__, ident, facet, operation};
+                        string message =
+                            createRequestFailedMessage("::Ice::ObjectNotExistException", ident, facet, operation);
+                        throw ObjectNotExistException{__FILE__, __LINE__, std::move(message), ident, facet, operation};
```

Comment:
We are missing a call to std::move for ident, facet, and operation.

Reply:
Fixed.